### PR TITLE
Update EventObject.after.dispatchEvent.html.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: dom/events/Event-defaultPrevented-after-dispatch.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Event.defaultPrevented is not reset after dipatchEvent()</title>
```
Comment (ayg) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: dipatch -> dispatch

File: dom/events/Event-defaultPrevented-after-dispatch.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id=log></div>
+<input id="target" type="hidden" value=""/>
```
Comment (ayg) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: Why is the event being fired at this element?  Why not fire at the document?

File: dom/events/Event-defaultPrevented-after-dispatch.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<script>
+test(function() {
+    var EVENT = "foo";
```
Comment (ayg) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: I think accepted style is not to have variables for named constants like this, and just to use the string literal.

File: dom/events/Event-defaultPrevented-after-dispatch.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    var TARGET = document.getElementById("target");
```
Comment (ayg) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: Variable names should be lowercase, I think.  (But there doesn't need to be a variable here if you fire at the document.)

**Comment by ayg** [comment]
LGTM without further changes (insofar as you're strictly improving the file), but even better if you address the review comments.

File: dom/events/Event-defaultPrevented-after-dispatch.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
```
Comment (ayg) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: Is there any particular reason to add this to files that contain only ASCII characters?

Comment (Ms2ger) [inline dom/events/Event-defaultPrevented-after-dispatch.html]: Just that it's a good habit in general.

**Comment by Ms2ger** [comment]
Thanks for the review, but I don't really feel like doing more work on this :)

**Pull request merged**